src/components/MovieDetails/index.tsx

```diff
@@ -356,18 +356,30 @@ const MovieDetails: React.FC<MovieDetailsProps> = ({ movie }) => {
             {data.title}{' '}
             <span className="text-2xl">({data.releaseDate.slice(0, 4)})</span>
           </h1>
-          <span className="mt-1 text-xs lg:text-base lg:mt-0">
+          <div className="mt-1 text-xs lg:text-base lg:mt-0">
```

Comment:
Why are we changing this to a `div`?

Reply:
I changed it to a `div` because I was building several `span`s under it and thus felt like a `div` was a more appropriate element. I'm changing it back to a `span` anyway because I'll be using the attributes array you made.

Reply:
I think that since `div` is a block element and `span` is inline, it makes more sense semantically to use `span` here since this is basically just a label.  No problem with nested `span` elements regardless 😉  Make sure to revert the change in `TvDetails` as well so it's consistent!

Reply:
On it!